webserver/cWebem.h

```diff
@@ -19,6 +19,12 @@ namespace http {
 			AUTH_LOGIN=0,
 			AUTH_BASIC,
 		};
+		enum _eWebCompressionMode
+		{
+			WWW_USE_GZIP=0,
```

Comment:
Could you make no-compression value 0 ?

Reply:
Is this enum public ? Then please use it everywhere (g_wwwCompressMode, in SetWebCompressionMode)

Reply:
It is public, but I need to transfer the value from domoticz.cpp to mainworker.cpp before I can use it. This is effectively the same method used for setting authentication method and webroot.

Reply:
There is actually no real logic to the enum. It merely seemed appropriate to use 0 for the default. If it is to be about (not) doing something special than the logical choice would be for `static` to be 0 as this is the setting that does not actively zip or unzip files.

Reply:
Correct, the Authentication method should also be an enum

0 = Off, or Disabled , not On or Enabled

Reply:
I think it is? The two available choices are actually auth_login and auth_basic.

Reply:
I know, but this is how it was supposed to do, and could inspire you to also use enums !